apps/common-app/src/examples/RuntimeTests/ReanimatedRuntimeTestsRunner/TestRunner.ts

```diff
@@ -585,4 +560,54 @@ export class TestRunner {
     }
     console.log('\n');
   }
+
+  private async mockConsole(testCase: TestCase) {
+    const counterUI = makeMutable(0);
+    let counterJS = 0;
+    const recordedMessage = makeMutable('');
+
+    const originalError = console.error;
+    const originalWarning = console.warn;
+
+    const incrementJS = () => {
+      counterJS++;
+    }
+    const consoleLogMock = (message: string) => {
```

Comment:
This function is not related with `console.log`, could we rename it a bit?
```suggestion
    const mockedConsoleFunction = (message: string) => {
```